.pre-commit-config.yaml

```diff
@@ -0,0 +1,61 @@
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.5.0
+  hooks:
+  - id: trailing-whitespace
+  - id: end-of-file-fixer
+  - id: check-yaml
+  - id: check-toml
+  - id: debug-statements
+- repo: https://github.com/macisamuele/language-formatters-pre-commit-hooks
+  rev: v2.12.0
+  hooks:
+  - id: pretty-format-yaml
+    args: [--preserve-quotes, --autofix, --indent, '2']
+  - id: pretty-format-toml
+    args: [--autofix, --indent, '4', --trailing-commas]
+- repo: https://github.com/pre-commit/pygrep-hooks
+  rev: v1.10.0
+  hooks:
+  - id: python-check-blanket-noqa
+  - id: python-check-blanket-type-ignore
+  - id: python-no-log-warn
+  - id: python-use-type-annotations
+  - id: rst-backticks
+  - id: rst-directive-colons
+  - id: rst-inline-touching-normal
+- repo: https://github.com/psf/black
+  rev: 24.2.0
+  hooks:
+  - id: black
+    args: [--safe, --quiet, --line-length=100]
+- repo: https://github.com/PyCQA/autoflake
+  rev: v2.3.0
+  hooks:
+  - id: autoflake
+    args: [--in-place, --remove-unused-variable]
+- repo: https://github.com/pycqa/isort
+  rev: 5.13.2
+  hooks:
+  - id: isort
+    name: isort
+    args: ["--force-single-line", "--line-length=100", "--profile=black"]
```

Comment:
No real argument. I guess it is because I was coming from https://github.com/asottile/reorder_python_imports . I like the `--force-single-line` for readability, though. I also think that a line-length of approx. 100 works well on modern screens.